backend/generate-makefile

```diff
@@ -0,0 +1,12 @@
+#!/bin/bash
```

Comment:
can this be toplevel?
what about `.sh`?

Reply:
i got rid of this